vault/login_mfa.go

```diff
@@ -295,6 +323,7 @@ func (i *IdentityStore) handleMFAMethodUpdateCommon(ctx context.Context, req *lo
 			ID:          configID,
 			Type:        methodType,
 			NamespaceID: ns.ID,
+			Name:        methodName,
 		}
 	}
 
```

Comment:
For clarity, my last question/suggestion would apply down here (after the pre-existing nil check).